src/components/NavBar.vue

```diff
@@ -61,52 +90,71 @@ export default Vue.extend({
       margin-top: 2.25rem;
     }
   }
-}
 
-.pointer {
-  cursor: pointer;
-}
-#profileIcon {
-  cursor: pointer;
-  background-image: url('~@/assets/images/navbar/profileIcon.svg');
-
-  &:hover,
-  &:focus,
-  &:active {
-    background-image: url('~@/assets/images/navbar/profileIconBlue.svg');
+  .profile-icon {
+    background-image: url('~@/assets/images/navbar/profileIcon.svg');
+
+    &:hover,
+    &:focus,
+    &:active {
+      background-image: url('~@/assets/images/navbar/profileIconBlue.svg');
+    }
+  }
+
+  .requirements-bar {
+    display: none;
+    background-image: url('~@/assets/images/navbar/requirement-toggle.svg');
   }
-}
 
-.requirementsBar {
-  cursor: pointer;
-  background-image: url('~@/assets/images/navbar/hamburger-gray.svg');
+  .hamburger {
+    display: none;
+    background-image: url('~@/assets/images/navbar/hamburger-gray.svg');
 
-  &:hover,
-  &:focus,
-  &:active {
-    background-image: url('~@/assets/images/navbar/hamburger-blue.svg');
+    &:hover,
+    &:focus,
+    &:active {
+      background-image: url('~@/assets/images/navbar/hamburger-blue.svg');
+    }
   }
-}
 
-#star {
-  cursor: pointer;
-  background-image: url('~@/assets/images/navbar/star.svg');
+  .logout-icon {
+    background-image: url('~@/assets/images/navbar/logout.svg');
 
-  &:hover,
-  &:focus,
-  &:active {
-    background-image: url('~@/assets/images/navbar/starBlue.svg');
+    &:hover,
+    &:focus,
+    &:active {
+      background-image: url('~@/assets/images/navbar/logoutBlue.svg');
+    }
+  }
+
+  .navbar-menu-background-shadow {
+    display: none;
+    position: fixed;
+    z-index: 2;
+    left: 0;
+    right: 0;
+    top: $mobile-navbar-height;
+    bottom: 0;
+    background: rgba(0, 0, 0, 0.5);
+  }
+
+  .navbar-menu {
+    position: fixed;
+    z-index: 3;
+    left: 0;
+    width: 16.5rem;
+    top: $mobile-navbar-height;
+    bottom: 0;
+    display: none;
+    background: $offWhite;
```

Comment:
I think the background color is off. In the designs, the background color is supposed to be `#F7FAFC`. `$offWhite` is `#F8F8F8`.